src/components/Editor/Runners/PythonRunner/PythonRunner.jsx

```diff
@@ -57,14 +57,15 @@ const PythonRunner = () => {
           );
           if (hasSkulptOnlyModules || senseHatAlwaysEnabled) {
             setUsePyodide(false);
+            break;
```

Comment:
Good solution
Why did you decide to put "break" here?

Reply:
This is to break out of looping over the components when one is found containing skulpt-only modules. The problem we were having only happened if are multiple components ("files"). If there was a file with skulpt-only imports it would switch to skulpt, but then if it found another file without skulpt-only imports afterwards it would switch back to pyodide and then the code wouldn't run. So we should break out of the loop as soon as we find a file with a skulpt-only import, as we already have enough information to know we need to use skulpt.